common/testData/personNameTest/_header.txt

```diff
@@ -0,0 +1,6 @@
+# Test data for unit conversions
```

Comment:
Unit conversions?

Reply:
Yeah, I just noticed that. I will make another PR to fix, sorry about that. But at least I can check the updated process that adds the header for catalog.txt etc. and verify that the copyright scan now passes in ICU.